tests/workflows/db/test_db_schemas.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from ophys_etl.workflows.db.schemas import OphysROI
+
+
+class TestOphysROI:
+
+    @pytest.mark.parametrize(
+        "is_valid,decrosstalk_none",
+        [(False, False), (True, False), (False, True), (True, True)])
+    def test_is_valid(self,
+                      is_valid: bool,
```

Comment:
clearer if you name this `is_in_motion_border` since it's a bit confusing when is_valid is True but decrosstalk_none is also True